website/docs/components/icon-tile/partials/guidelines/guidelines.md

```diff
@@ -23,7 +23,122 @@ Use a **product-specific color** for objects or pages that are directly related
 
 ### Examples
 
-
+TEST, horizontal:
```

Comment:
I added all these temporary examples to demonstrate the Doc::Layout component. They will be removed before merging.